requirements.txt

```diff
@@ -12,3 +12,4 @@ requests
 paramiko
 psutil>=5.5.1
 setproctitle
+autopep8
```

Comment:
what is this for?

Reply:
I found where it was used - sorry.

Maybe consider adding taskvine-specific dependencies to setup.py, in `extras_require` with a `taskvine` target: so that users would install like this:

`pip install 'parsl[taskvine]'`

Reply:
autopep8 is no longer needed and is removed.